client_request.go

```diff
@@ -134,7 +134,7 @@ func (c *Client) sendRequest(req *internalRequest, internalError *Error, respons
 		request.Header.Set("Content-Type", req.contentType)
 	}
 	if c.config.APIKey != "" {
-		request.Header.Set("Authorization", "Bearer " + c.config.APIKey)
+		request.Header.Set("Authorization", "Bearer "+c.config.APIKey)
```

Comment:
The formater prefers no whitespaces around the `+` sign?

Reply:
Me too, but the linter disagrees with us ...